### ISSUE TITLE
Unable to run on Kubuntu 18.10

### ISSUE THREAD

**Issue by gorbeia** [Artanidos/AnimationMaker#7]
I have downloaded the appimage file and the application does not run on Kubuntu 18.10.

I get the following output

```
$ export QT_DEBUG_PLUGINS=1
$ ./AnimationMaker-Linux-x86_64-1.8.2.AppImage 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_AnimatJgRyMT/usr/bin/platforms" ...
This application failed to start because it could not find or load the Qt platform plugin "xcb"
in "".

Reinstalling the application may fix this problem.
Aborted (core dumped)
```

**Comment by dmbomer** [comment]
The program just does run on my system no error output on the screen but when I check the system log
 2 09:30:03 deepcool org.freedesktop.FileManager1[1538]: This application failed to start because it could not find or load the Qt platform plugin "xcb"
Mar  2 09:30:03 deepcool org.freedesktop.FileManager1[1538]: in "".
Mar  2 09:30:03 deepcool org.freedesktop.FileManager1[1538]: Reinstalling the application may fix this problem.
I am using the appimage 1.8.2 https://www.linux-apps.com/p/1193218/
My system is Ubuntu 18.04
Linux deepcool 4.15.0-45-generic #48-Ubuntu SMP Tue Jan 29 16:28:13 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
AMD FX 8230
32gb memory
240 ssd
RX570 4gb
What am I reinstalling, I'm confused.

**Comment by frank4360** [comment]
I am running on a laptop with x86_64 processor, 4 gb RAM,
software:
Kernel		: Linux 4.9.0-8-amd64 (x86_64)
Distribution		: Debian GNU/Linux 9.8

and have the same problem with 
AnimationMaker-Linux-x86_64-1.8.2.AppImage.

**The error message is :
This application failed to start because it could not find or load the Qt platform plugin "xcb"
in "".

Reinstalling the application may fix this problem.
Aborted**

I have tried reloading and also using earlier versions of the package - always same result.
I have used AnimationMaker previously without a problem, but possibly on an earlier version of Debian.
